src/richText.js

```diff
@@ -232,6 +232,15 @@ define([
                 editor.fire("saveSnapshot");
                 return wrapper;
             },
+            focus: function() {
+                if (editor.status === "ready") {
+                    editor.focus();
+                } else {
+                    editor.on('instanceReady', function () {
```

Comment:
Good point. Updated to turn the listener off before adding it in https://github.com/dimagi/Vellum/pull/808/commits/eded8b679928103b330b18592074bc0ba19dea1b (just because it's less code than adding a flag).